src/app/redux/GlobalReducer.test.js

```diff
@@ -0,0 +1,719 @@
+import chai, { expect } from 'chai';
+import chaiImmutable from 'chai-immutable';
+
+import { Map, OrderedMap, getIn, List, fromJS, Set, merge } from 'immutable';
+import { emptyContent } from 'app/redux/EmptyState';
+
+import * as globalActions from './GlobalReducer';
+import reducer, { defaultState } from './GlobalReducer';
+
+chai.use(chaiImmutable);
+
+const expectedStats = Map({
+    isNsfw: false,
+    hide: false,
+    hasPendingPayout: false,
+    gray: false,
+    flagWeight: 0,
+    up_votes: 0,
+    total_votes: 0,
+    authorRepLog10: undefined,
+    allowDelete: true,
+});
+
+const mockPayloads = {
```

Comment:
should we define these payloads at the top like this, or define them inline with each test? having them inline makes each test a bit easier to read.